### DISCUSSION TITLE
Am I using the large disk version?

### DISCUSSION THREAD

**Discussion by gqqnbig** [seaweedfs/seaweedfs#3106] [General]
I don't know the version I downloaded is large_disk or not. How would I check it?

```console
$ ./weed version
version 8000GB 3.06 2f846777bbceea307771e79d4452e071b0bd5a51 linux amd64
$ sudo ./weed master -mdir=weed-idx -ip=172.25.17.14
I0526 20:55:48  6128 file_util.go:23] Folder weed-idx Permission: -rwxrwxr-x
I0526 20:55:48  6128 master.go:255] current: 172.25.17.14:9333 peers:
I0526 20:55:48  6128 master_server.go:131] Volume Size Limit is 30000 MB
I0526 20:55:48  6128 master.go:147] Start Seaweed Master 8000GB 3.06 2f846777bbceea307771e79d4452e071b0bd5a51 at 172.25.17.14:9333
```

The log says " Volume Size Limit is 30000 MB". Does it mean my version is small disk version?

Note this is the first time I ever use seaweed, I don't think I need to follow  the "To upgrade to large disk version" part in https://github.com/chrislusf/seaweedfs/wiki/FAQ#how-to-configure-volumes-larger-than-30gb 

Does "Start Seaweed Master 8000GB" in the log mean the total overall storage size is only 8TB? 

How do I achieve the 128 exbibytes mentioned on https://github.com/chrislusf/seaweedfs#storage-size ?

**Comment by kmlebedev** [comment]
8000GB - is large disk version
But
  -volumeSizeLimitMB uint
    	Master stops directing writes to oversized volumes. (default 30000)

> Does "Start Seaweed Master 8000GB" in the log mean the total overall storage size is only 8TB?

No, This means that the maximum available size for one super block out of millions
